examples/md-flexible/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 file(GLOB_RECURSE MY_SRC
         "*.cpp"
         "*.h"
+        "../../tests/testAutopas/testingHelpers/RandomGenerator.cpp"
```

Comment:
Eventually we should move this to some proper location, i.e., utils directory of autopas or some extra directory outside of autopas.

Reply:
and all other generators

Reply:
#129